src/workerd/api/http.c++

```diff
@@ -920,10 +917,7 @@ jsg::Ref<Request> Request::constructor(
 jsg::Ref<Request> Request::clone(jsg::Lock& js) {
   auto headersClone = headers->clone();
 
-  auto cfClone = cf.map([&](jsg::V8Ref<v8::Object>& obj) {
```

Comment:
@jasnell any idea why did we need deepClone here? Maybe only one clone is enough?